### PR TITLE
Added uberpeople.net.

### PR DESCRIPTION
**Pull request by awelch83**

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -175,6 +175,17 @@ websites:
       software: Yes
       doc: https://support.twitter.com/articles/20170388
 
+    - name: Uberpeople.net
+      url: http://uberpeople.net
+      img: site.png
```
Comment (dsoegijono) [inline _data/social.yml]: This is incorrect image name, should be `uberpeople.png`

**Comment by Carlgo11** [comment]
Hello @awelch83,
I'd just like to point out that the build log of your PR, along with error messages, can be found over at [Travis-CI](https://travis-ci.org/2factorauth/twofactorauth/builds/141126965).

The current build log gives these error messages:

```
<------------ ERROR ------------>

1. Uberpeople.net should not contain a 'hardware' tag when it's value isn't 'YES'.

2. Uberpeople.net should not contain a 'sms' tag when it's value isn't 'YES'.

3. Uberpeople.net should not contain a 'phone' tag when it's value isn't 'YES'.

4. img/social/uberpeople.png is not 32x32

5. img/social/uberpeople.png should not be larger than 2500 bytes. It is currently 19313 bytes
```

Along with that I'd also like to point out that the `doc:` tag isn't needed if you don't have any link to redirect to. :smile: 

//Carl

**Comment by awelch83** [comment]
@Carlgo11 Thanks for taking the time to get me straightened out. I look forward to making additional contributions

**Comment by Carlgo11** [comment]
Great! Here's a smaller image. Even though the one you used works this one is 40% smaller. :wink: 
uberpeople

**Pull request closed without merging**